stdlib/builtin/string_io.rbs

```diff
@@ -6,4 +6,8 @@ class StringIO
   def write: (String) -> void
   def flush: () -> void
   def string: -> String
+  def close_read: -> NilClass
```

Comment:
Could you use `nil` in this case?

There are some out-of-date `NilClass` uses in existing signatures and I'm sorry for confusion. We prefer writing `nil` usually.

Reply:
Thank you for reviewing this PR!!

Okay. I fixed it at https://github.com/ruby/ruby-signature/pull/71/commits/b6cdb488be324c239f7031877f927fea95032bdd .
Please let me know if commits in this PR should be squashed into one commit :)

Reply:
@MITSUBOSHI Rebasing and squashing trivial fixes are preferred, if not too difficult! 👋

Reply:
@soutaro I did! Could you please review again?

Reply:
Thanks!